components/user/UserTabs.js

```diff
@@ -2,152 +2,158 @@ import { BiSortAlt2 } from "react-icons/bi";
-        ? getNested(a, sortKey.split("."))
-        : a[sortKey];
-      const bVal = sortKey.includes(".")
-        ? getNested(b, sortKey.split("."))
-        : b[sortKey];
-      if (tabName === "My Links") {
-        if (order === "ASC") {
-          return aVal.toLowerCase() > bVal.toLowerCase()
-            ? 1
-            : aVal.toLowerCase() < bVal.toLowerCase()
-            ? -1
-            : 0;
-        } else {
-          return aVal.toLowerCase() < bVal.toLowerCase()
-            ? 1
-            : aVal.toLowerCase() > bVal.toLowerCase()
-            ? -1
-            : 0;
-        }
-      } else {
-        if (order === "ASC") {
-          return new Date(aVal) > new Date(bVal)
-            ? 1
-            : new Date(aVal) < new Date(bVal)
-            ? -1
-            : 0;
-        } else {
-          return new Date(aVal) < new Date(bVal)
-            ? 1
-            : new Date(aVal) > new Date(bVal)
-            ? -1
-            : 0;
-        }
-      }
-    });
-    setUserData({ ...userData });
-  };
+ const sortUserTabItems = (tabName, order) => {
+  const { dataKey, sortKey } = getDataKeyAndSortKey(tabName);
+  userData[dataKey].sort(function (a, b) {
+   const aVal = sortKey.includes(".")
+    ? getNested(a, sortKey.split("."))
+    : a[sortKey];
+   const bVal = sortKey.includes(".")
+    ? getNested(b, sortKey.split("."))
+    : b[sortKey];
+   if (tabName === "My Links") {
+    if (order === "ASC") {
+     return aVal.toLowerCase() > bVal.toLowerCase()
+      ? 1
+      : aVal.toLowerCase() < bVal.toLowerCase()
+      ? -1
+      : 0;
+    } else {
+     return aVal.toLowerCase() < bVal.toLowerCase()
+      ? 1
+      : aVal.toLowerCase() > bVal.toLowerCase()
+      ? -1
+      : 0;
+    }
+   } else {
+    if (order === "ASC") {
+     return new Date(aVal) > new Date(bVal)
+      ? 1
+      : new Date(aVal) < new Date(bVal)
+      ? -1
+      : 0;
+    } else {
+     return new Date(aVal) < new Date(bVal)
+      ? 1
+      : new Date(aVal) > new Date(bVal)
+      ? -1
+      : 0;
+    }
+   }
+  });
+  setUserData({ ...userData });
+ };
 
-  const getNested = (obj, args) => {
-    return args.reduce((obj, level) => obj && obj[level], obj);
-  };
+ const getNested = (obj, args) => {
+  return args.reduce((obj, level) => obj && obj[level], obj);
+ };
+ console.log(tabs);
+ return (
+  <div>
+   <div className="sm:hidden">
+    {tabs.length > 1 ? (
+     <>
+      <label htmlFor="tabs" className="sr-only">
+       Select a tab
+      </label>
+      <select
+       id="tabs"
+       name="tabs"
+       onChange={changeTab}
+       className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
+       defaultValue={tabs.find((tab) => tab.current).name}
+      >
+       {tabs.map((tab) => (
+        <option key={tab.name}>{tab.name}</option>
+       ))}
+      </select>
+     </>
+    ) : (
+     <h3 className="font-medium border-b-2 text-center">
```

Comment:
Also add some padding on here, the border under looks too close

Reply:
Sure